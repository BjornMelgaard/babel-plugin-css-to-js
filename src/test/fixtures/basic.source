
csjs`.foo{ transform: ${foo}; color: ${bar} }`;
