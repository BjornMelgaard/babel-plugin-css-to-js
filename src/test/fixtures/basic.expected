
csjs`.foo{ -webkit-transform: ${ foo }; transform: ${ foo }; color: ${ bar } }`;
